.github/workflows/publish.yml

```diff
@@ -15,7 +15,6 @@ jobs:
     uses: babylonlabs-io/.github/.github/workflows/reusable_go_lint_test.yml@v0.7.0
     with:
       run-build: true
-      run-lint: true
```

Comment:
why do we remove this?

Reply:
Honestly as it didn't work, this github job just get's stuck without errors, and as it's importing this action from babylon. I didn't want to waste time but will check again

Reply:
I see, I think this should be easy fix, but np if we comment it out for now